requirements.txt

```diff
@@ -1,21 +1,23 @@
-aiohttp>=3.6.2
-async-timeout>=3.0.1
-attrs>=19.3.0
-beautifulsoup4>=4.9.1
-bs4>=0.0.1
-certifi>=2020.6.20
-chardet>=3.0.4
-discord.py>=1.5.1
-idna>=2.10
-lavalink>=3.1.2
-multidict>=4.7.6
-mysql-connector>=2.2.9
-psutil>=5.7.2
-python-dotenv>=0.14.0
-requests>=2.24.0
-soupsieve>=2.0.1
-typing-extensions>=3.7.4.2
-urllib3>=1.25.10
-websockets>=8.1
-yarl>=1.5.1
-mysql-connector-python>=8.0.21
+aiohttp==3.6.3
```

Comment:
upgrade aiohttp to 3.7.4+ to fix the 'Open redirect' vulnerabilities

Reply:
:eyes: